tools/docker-network/docker-compose.yml

```diff
@@ -245,6 +235,61 @@ services:
       --faucet.bindAddress=inx-faucet:8091
       --faucet.rateLimit.enabled=false
 
+  inx-validator-1:
+    image: iotaledger/inx-validator:latest
+    stop_grace_period: 1m
+    restart: unless-stopped
+    depends_on:
+      node-1-validator:
+        condition: service_started
+      inx-indexer:
+        condition: service_started
+    environment:
+      - "VALIDATOR_PRV_KEY=443a988ea61797651217de1f4662d4d6da11fd78e67f94511453bf6576045a05293dc170d9a59474e6d81cfba7f7d924c09b25d7166bcfba606e53114d0a758b"
+    command:
```

Comment:
Can you use the multiline syntax with `command: >`. Andrea wanted to have it that way for the rest as well.